### PR TITLE
UIFeature/ui admin error handling fix

### PR DESCRIPTION
**Pull request by astewes**
*  Adds new alert styling to admin states (including modals)
*  Consolidates alert styling

TO DO:
-  [x] Resolve issue where error in create namespace modal displays both as a ribbon and in the global notification popover

app

namespace

### CODE REVIEW

File: cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js [hunk 1]
```diff
@@ -40,9 +40,9 @@ angular.module(PKG.name + '.feature.admin')
         .then(
           function success(res) {
             $scope.isSaving = false;
-            $alert({
-              content: res,
-              type: 'success'
+            myAlert({
+              type: 'success',
+              content: res
```
Comment (elfenheart) [inline cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js]: I don't think this success message should go to myAlert

File: cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js [hunk 2]
```diff
@@ -54,6 +54,11 @@ angular.module(PKG.name + '.feature.admin')
           function error(err) {
             $scope.isSaving = false;
             $scope.error = err.data;
+            myAlert({
+              type: 'danger',
+              content: $scope.error,
+              duration: false
+            });
```
Comment (elfenheart) [inline cdap-ui/app/features/admin/controllers/namespace/create-ctrl.js]: this myAlert is what's causing it to show on the global error message

**Comment by elfenheart** [comment]
Missed 1 thing:
On setting instance under Services > Go into any of the service. You can set an instance there. I believe that is a directive 'instance-control'

File: cdap-ui/app/directives/instance-control/instance-control.js [hunk 3]
```diff
@@ -40,9 +40,19 @@ angular.module(PKG.name + '.commons')
...
-      });
+      }).then(
+        function success () {
+          $scope.instance.provisioned = $scope.instance.requested;
+        },
+        function error(err) {
+          myAlertOnValium.show({
+            type: 'danger',
+            title: 'Error',
+            content: err,
```
Comment (elfenheart) [inline cdap-ui/app/directives/instance-control/instance-control.js]: let's change this err to ```angular.isObject(err) ? err.data : err ```

**Comment by elfenheart** [comment]
:+1: LGTM for @astewes changes

**Comment by ajainarayanan** [comment]
:+1: LGTM. Tested it locally and the error surfacing works fine.

**Pull request merged**